inference-engine/src/gna_plugin/backend/make_pwl.cpp

```diff
@@ -223,7 +224,7 @@ void make_gna_pwl(const DnnActivation  fun,
             else
                 gnalog() << "=========================== LeakyReLU Segments ======================\n";
             int32_t x_lower = INT32_MIN;
-            int16_t y_lower = INT16_MIN;
+            int16_t y_lower = low_precision ? INT8_MIN : INT16_MIN;
```

Comment:
I'm a little bit confused by this change. What are you trying to do?
Is this change enabling dumping models or enables int8 activations?

Reply:
Dumping models is just a part of this change, to allow more convenient debugging.
The idea of the whole PR is to enable low precision (int8) path.